wiki/History_of_osu!/Online_rankings/osu!catch/en.md

```diff
@@ -0,0 +1,100 @@
+| ::{ flag=KR }:: [Motion](https://osu.ppy.sh/users/3885626) | 2019-04-06 | 2019-08-09 | [\[1\]](https://web.archive.org/web/20190406161641/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549) | 2019-08-09 | 2019-08-18 | [\[1\]](https://www.reddit.com/r/osugame/comments/co0ruo/after_2_years_224hyperw7_has_reclaimed_rank_1_on/) [\[2\]](https://web.archive.org/web/20190810115254/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [Motion](https://osu.ppy.sh/users/3885626) | *2019-08-18* | *2020-02-04* | [\[1\]](https://web.archive.org/web/20190818051437/https://osu.ppy.sh/rankings/fruits/performance) |
+
+## 2020
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ::{ flag=KR }:: [Motion](https://osu.ppy.sh/users/3885626) | *2019-08-18* | 2020-02-04 | [\[1\]](https://web.archive.org/web/20190818051437/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=HK }:: [AutoLs](https://osu.ppy.sh/users/9155988) | 2020-02-04 | 2020-05-14 | [\[1\]](https://web.archive.org/web/20200204173040/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [Motion](https://osu.ppy.sh/users/3885626) | *2020-05-14* | *2021-01-12* | [\[1\]](https://web.archive.org/web/20200514200624/https://osu.ppy.sh/rankings/fruits/performance) |
+
+## 2021
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ::{ flag=KR }:: [Motion](https://osu.ppy.sh/users/3885626) | *2020-05-14* | 2021-01-12 | [\[1\]](https://web.archive.org/web/20200514200624/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=HK }:: [AutoLs](https://osu.ppy.sh/users/9155988) | 2021-01-12 | 2021-04-14 | [\[1\]](https://web.archive.org/web/20210112170723/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549) | 2021-04-14 | 2021-04-22 | [\[1\]](https://web.archive.org/web/20210414210651/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=HK }:: [AutoLs](https://osu.ppy.sh/users/9155988) | 2021-04-22 | 2021-04-27 | [\[1\]](https://web.archive.org/web/20210422102910/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549) | 2021-04-27 | 2021-04-29 | [\[1\]](https://web.archive.org/web/20210427172037/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=HK }:: [AutoLs](https://osu.ppy.sh/users/9155988) | 2021-04-29 | 2021-05-05 | [\[1\]](https://web.archive.org/web/20210429090335/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549) | *2021-05-05* | *2024-07-21* | [\[1\]](https://web.archive.org/web/20230308031543/https://osu.ppy.sh/wiki/en/History_of_osu!/Online_rankings) [\[2\]](https://web.archive.org/web/20210608065616/https://osu.ppy.sh/users/4158549) [\[3\]](https://web.archive.org/web/20210505073934/https://osu.ppy.sh/rankings/fruits/performance) |
+
+## 2024
```

Comment:
if just copying the osu page then there should be sections for 2022 and 2023 which contain a continued listing for yesmydarkness (e.g. https://osu.ppy.sh/wiki/en/History_of_osu%21/Online_rankings/osu%21#2017)